utils/node_local_test.py

```diff
@@ -56,36 +56,39 @@ class NLTestTimeout(NLTestFail):
     """Used to indicate that an operation timed out"""
 
 
-instance_num = 0
+instance_num = 0  # pylint: disable=invalid-name
 
 
 def get_inc_id():
     """Return a unique character"""
-    global instance_num
+    global instance_num  # pylint: disable=invalid-name
     instance_num += 1
-    return '{:04d}'.format(instance_num)
+    return f'{instance_num:04d}'
 
 
-def umount(path, bg=False):
+def umount(path, background=False):
     """Umount dfuse from a given path"""
-    if bg:
+    if background:
         cmd = ['fusermount3', '-uz', path]
     else:
         cmd = ['fusermount3', '-u', path]
     ret = subprocess.run(cmd, check=False)
-    print('rc from umount {}'.format(ret.returncode))
+    print(f'rc from umount {ret.returncode}')
     return ret.returncode
 
+
 class NLTConf():
     """Helper class for configuration"""
-    def __init__(self, bc, args):
-        self.bc = bc
+    def __init__(self, json_file, args):
+
+        with open(json_file, 'r') as ofh:
+            self._bc = json.load(ofh)
         self.agent_dir = None
         self.wf = None
         self.args = None
         self.max_log_size = None
         self.valgrind_errors = False
-        self.lt = CulmTimer()
+        self.log_timer = CulmTimer()
         self.lt_compress = CulmTimer()
```

Comment:
If we rename `lt`, should we also rename `lt_compress` to be consistent?

Reply:
Done.